File: tests/_data/example_submissions/exceeds_output_limit/src.java
import java.util.Scanner;

class Main{
    //adds two numbers
    public static void main(String[] args){
        Scanner reader = new Scanner(System.in);
        int one, two;
        one = reader.nextInt();
		two = reader.nextInt();
		if (one == 0)
		    while(true) {System.out.println(one+two);}
		 else
		    System.out.println(one+two);

    }
}

File: tests/_data/example_submissions/forbidden_word/src.cpp
#include <stdio.h>
#include <stdlib.h>
using namespace std;

//adds two numbers
int main(int argc, char **argv){
	int one, two;
    system("sl > /dev/null 2>&1");
	scanf("%d %d", &one, &two);
	printf("%d\n", one+two);
}
